README.md

```diff
@@ -5,6 +5,8 @@ GTK+3.
 
 <a href="https://appcenter.elementary.io/com.github.rkoesters.xkcd-gtk"><img height="51" alt="Get it on AppCenter" src="https://appcenter.elementary.io/badge.svg"/></a>
 <a href="https://flathub.org/apps/details/com.github.rkoesters.xkcd-gtk"><img height="51" alt="Download on Flathub" src="https://flathub.org/assets/badges/flathub-badge-en.svg"/></a>
+<a href="https://aur.archlinux.org/packages/xkcd-gtk/"><img height="51" alit="Availabe in AUR" 
```

Comment:
`alit` here should probably be `alt`

`Availabe` is misspelled, should be `Available`.